com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/EmissionUIBlock.cs

```diff
@@ -9,14 +9,22 @@
 
 namespace UnityEditor.Rendering.HighDefinition
 {
-    class EmissionUIBlock : MaterialUIBlock
+    /// <summary>
+    /// Emission material UI block.
+    /// </summary>
+    public class EmissionUIBlock : MaterialUIBlock
     {
+        /// <summary>Feature of the Emission block, used to control which field is visible.</summary>
```

Comment:
Options for emission block features. Use this to control which properties are visible.